crypto/ec/ec_lib.c

```diff
@@ -1025,3 +1025,11 @@ int EC_GROUP_do_inverse_ord(const EC_GROUP *group, BIGNUM *res,
     else
         return 0;
 }
+
+int ec_point_blind_coordinates(const EC_GROUP *group, EC_POINT *p, BN_CTX *ctx)
+{
+    if (!group->meth->blind_coordinates)
```

Comment:
Check for NULL.

Reply:
fixed !

Reply:
We introduced a small bug fixing this.
Now it is fixed for real ;)